app/src/main/java/openfoodfacts/github/scrachx/openfood/views/SaveProductOfflineActivity.java

```diff
@@ -88,7 +103,11 @@ protected void onCreate(Bundle savedInstanceState) {
         getSupportActionBar().setDisplayHomeAsUpEnabled(true);
 
         mSendProductDao = Utils.getAppDaoSession(this).getSendProductDao();
-
+        mSharedPref = getApplicationContext().getSharedPreferences("prefs", 0);
+        boolean messageDismissed = mSharedPref.getBoolean("is_message_dismissed", false);
```

Comment:
Can we be much more explicit about the message ? I expect that eventually we'll have many explainer messages at many places in the app.